oxidation/libparsec/crates/client_connection/src/error.rs

```diff
@@ -67,6 +67,43 @@ pub enum CommandError {
     WrongApiVersion(String),
 }
 
+impl PartialEq for CommandError {
+    fn eq(&self, other: &Self) -> bool {
+        match (self, other) {
+            (Self::BadContent, Self::BadContent) => true,
+            (Self::ExpiredOrganization, Self::ExpiredOrganization) => true,
+            // For the moment, InvalidResponseContentError are the same
+            (Self::InvalidResponseContent(..), Self::InvalidResponseContent(..)) => true,
+            // For the moment, InvalidResponseStatus are the same if they have the same status
+            (
+                Self::InvalidResponseStatus(status0, ..),
+                Self::InvalidResponseStatus(status1, ..),
+            ) => status0 == status1,
+            (Self::InvitationAlreadyDeleted, Self::InvitationAlreadyDeleted) => true,
+            (Self::InvitationNotFound, Self::InvitationNotFound) => true,
+            (Self::MissingApiVersion, Self::MissingApiVersion) => true,
+            (Self::MissingSupportedApiVersions, Self::MissingSupportedApiVersions) => true,
+            // For the moment, NoResponseError are the same
+            (Self::NoResponse(..), Self::NoResponse(..)) => true,
+            (Self::RevokedUser, Self::RevokedUser) => true,
+            // For the moment, SerializationError are the same
+            (Self::Serialization(..), Self::Serialization(..)) => true,
+            (
+                Self::UnsupportedApiVersion {
+                    api_version: v0, ..
```

Comment:
v0 vs v1 is misleading (it seems you are refering to API version v0 vs v1, where it left version vs right version)

replace it by `right_api_version/left_api_version` ?

Reply:
Done